### PR TITLE
Image delta client support with progress reporting

### PR DESCRIPTION
**Pull request by petrosagg**
https://asciinema.org/a/Qlk0tgTAE8f8yv0yxImLWkLPV

### CODE REVIEW

**Comment by petrosagg** [comment]
oops. no, I accidentally added them in the commit. good catch!

**Comment by petrosagg** [comment]
fixed

File: api/swagger.yaml [hunk 1]
```diff
@@ -5238,7 +5238,7 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
       tags: ["Image"]
-  /deltas/create:
+  /image/delta:
```
Comment (dfunckt) [inline api/swagger.yaml]: Shouldn't this be `/images/delta`? Either that or `client/image_delta.go:17` is correct, can't be both :P

File: daemon/create.go [hunk 2]
```diff
@@ -341,42 +343,63 @@ func (daemon *Daemon) verifyNetworkingConfig(nwConfig *networktypes.NetworkingCo
...
+	progress.Update(progressOutput, deltaSrc, "Fingerprint complete")
+
 	deltaRootFS := image.NewRootFS()
 
+	for _, diffID := range dstImg.RootFS.DiffIDs {
+		progress.Update(progressOutput, stringid.TruncateID(diffID.String()), "Waiting")
+	}
+
+	statTotalSize := int64(0)
+	statDetlaSize := int64(0)
```
Comment (dfunckt) [inline daemon/create.go]: Typo: *detla*

**Pull request merged**